tools/topology/topology2/include/pipelines/cavs/highpass-capture-be.conf

```diff
@@ -0,0 +1,93 @@
+#
+# BE DAI highpass pipeline for capture
+#
+# All attributes defined herein are namespaced by alsatplg to
+# "Object.Pipeline.highpass-be-capture.N.attribute_name"
+#
+# Usage: highpass-capture-be pipeline object can be instantiated as:
+#
+# Object.Pipeline.highpass-capture-be."N" {
+#	direction	"capture"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."highpass-capture-be" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			type dai_in
+			num_audio_formats 1
+			num_input_audio_formats 1
+			num_output_audio_formats 1
+
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$obs * 2]"
+			}
+		}
+
+		eqiir."1" {
+			num_audio_formats 2
+			num_input_audio_formats 2
+			num_output_audio_formats 2
+
+			# 32-bit 48KHz 2ch
```

Comment:
The 24 bit format seems to be unnecessary since the above copier is 32 bits only.

Reply:
Fixed now